x-pack/packages/ml/is_defined/index.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export { MlStorageContextProvider, useStorage } from './storage_context';
+export { isDefined } from './src/is_defined';
```

Comment:
Could you add a comment to this exported function to keep the 'public APIs without comments' count at 0?